scripts/tools/run-thunder-command.js

```diff
@@ -148,5 +160,22 @@ switch (args.command) {
       handleResponse(err, result, 'DELETE');
     });
 
+    break;
+  case 'email':
+    hashedPassword = crypto.createHash('md5')
```

Comment:
So I know it's only a test but is this hash used to send real passwords in production?

Reply:
No this isn't used today, although presumably anyone could run this and use it to send real data to a production instance of Thunder. I do have a task to use a better hash in the `run-thunder-command`, I will clean this up soon.